services/backend/services/v4/entrypoints/db_migration.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "MONGODB_URI=$MONGODB_URI" >.env
```

Comment:
why is this needed?

Reply:
because the migration script uses the `dotenv` npm package which reads env variables from a `.env` file